### PR TITLE
siege: 4.0.8 -> 4.0.9

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 4.0.9

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by dotlambda — COMMENTED** [review]
Can we add https://www.joedog.org/siege-home/ as homepage?

**Review by diogox — APPROVED** [review]

**Comment by fabaff** [comment]
> Can we add https://www.joedog.org/siege-home/ as homepage?

Thanks, added.

File: pkgs/tools/networking/siege/default.nix [hunk 1]
```diff
@@ -32,8 +32,9 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "HTTP load tester";
+    homepage = "    homepage = "https://github.com/Instagram/libcst";";
```
Comment (dotlambda) [inline pkgs/tools/networking/siege/default.nix:35]:
```suggestion
    homepage ="https://www.joedog.org/siege-home/";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/tools/networking/siege/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/networking/siege/default.nix]: Ups, that went completely wrong.

**Review by dotlambda — APPROVED** [review]

**Pull request merged**